odoo/fields.py

```diff
@@ -335,6 +335,7 @@ class Second(models.Model):
 
         'related_field': None,          # corresponding related field
         'group_operator': None,         # operator for aggregating values
+        'group_full': None,             # name of method to retrieve all groups
```

Comment:
What about `group_extend` or `group_expand` ? Using a verb also conveys better that this attribute is a method name, like `compute` ...